packages/ui/src/accounts/hooks/useIsVoteStakeLocked.ts

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from 'react'
+
+import { LatestElection } from '@/council/types/LatestElection'
+import { useIsCouncilMember } from '@/memberships/hooks/useIsCouncilMember'
+import { Member } from '@/memberships/types'
+
+export const useIsVoteStakeLocked = (latestElection?: LatestElection, candidate?: Member) => {
+  const isCouncilMember = useIsCouncilMember(candidate)
```

Comment:
Since #1536 was closed I think there's no need for this hook anymore. The `false` in `@/memberships/types/casting.ts` should just be replaced by `data.isCouncilMember` and `useIsCouncilMember` should be remove (I should have done this in #1990 though :facepalm:)